src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -0,0 +1,332 @@
+
+###1. Update the transaction process and remove Stripe related actions
+
+You need to remove at least the following actions:
+
+- :action/stripe-create-payment-intent
+- :action/stripe-capture-payment-intent
+- :action/stripe-confirm-payment-intent
+- :action/stripe-create-payout
+- :action/stripe-refund-charge
+- :action/stripe-refund-payment
+
+If you are using the FTW default process with strong customer authentication
+(e.g. flex-default-process) you need to remove confirming the payment.
+Otherwise, the transaction will get stuck. The simplest way to do this is to
+remove `pending-payment` and `expire-payment` states and point `request-payment`
+and `equest-payment-after-enquiry` directly to `preauthorized` state. However,
+when you are building the transaction process to your marketplace it might make
+sense to rename some of the states and transitions (e.g. preauthorized) so that
+they make more sense in your use-case.
+
+#### Edit util/transaction.js to reflect the changes in your transaction process
+
+In
+[src/util/transaction.js](https://github.com/sharetribe/flex-template-web/blob/master/src/util/transaction.js)
+file, there are lots of helper functions that are used to determine which state
+the transaction is. This file should be updated to match the new transaction
+process.
+[Read more about editing transaction.js](/cookbook-transaction-process/change-transaction-process-in-ftw/#2-check-if-the-transactionjs-file-needs-to-be-updated)
+
+Example:
+
+```diff
+     [STATE_INITIAL]: {
+       on: {
+         [TRANSITION_ENQUIRE]: STATE_ENQUIRY,
+-        [TRANSITION_REQUEST_PAYMENT]: STATE_PENDING_PAYMENT,
++        [TRANSITION_REQUEST_PAYMENT]: STATE_PREAUTHORIZED,
+       },
+     },
+     [STATE_ENQUIRY]: {
+       on: {
+-        [TRANSITION_REQUEST_PAYMENT_AFTER_ENQUIRY]: STATE_PENDING_PAYMENT,
++        [TRANSITION_REQUEST_PAYMENT_AFTER_ENQUIRY]: STATE_PREAUTHORIZED,
+       },
+     },
+-
+-    [STATE_PENDING_PAYMENT]: {
+-      on: {
+-        [TRANSITION_EXPIRE_PAYMENT]: STATE_PAYMENT_EXPIRED,
+-        [TRANSITION_CONFIRM_PAYMENT]: STATE_PREAUTHORIZED,
+-      },
+-    },
+-
+-    [STATE_PAYMENT_EXPIRED]: {},
+     [STATE_PREAUTHORIZED]: {
+       on: {
+         [TRANSITION_DECLINE]: STATE_DECLINED,
+
+```
+
+###2. Remove Stripe checks from EditListingWizard
+
+In
+[`EditListingWizard.js`](https://github.com/sharetribe/ftw-daily/blob/master/src/components/EditListingWizard/EditListingWizard.js)
+we are checking if the provider has a Stripe account with all the required
+information before we allow them to publish listings. This check is done in the
+`handlePublishListing` function and needs to be removed.
+
+```js
+  handlePublishListing(id) {
+    const { onPublishListingDraft} = this.props;
+    onPublishListingDraft(id);
+  }
+```
+
+After changing the `handlePublishListing` you can remove unused code. You also
+might want to clean up `EditListingPage` and remove the Stripe related props we
```

Comment:
Is there any link to the correct lines (where to begin this cleaning) or is there too many of them?

Reply:
On `EditLisitngPage` there are mostly props that are passed forward and some variables related to Stripe so not very easy to point to just one line.